subscription-manager.spec

```diff
@@ -514,43 +526,25 @@ subscriptions
 %endif
 
 
-%if %use_dnf
-%package -n dnf-plugin-subscription-manager
-Summary: Subscription Manager plugins for DNF
+%if %{use_libdnf}
+%package -n libdnf-plugin-subscription-manager
+Summary: Subscription Manager plugin for libdnf
 %if 0%{?suse_version}
 Group: Productivity/Networking/System
 %else
 Group: System Environment/Base
 %endif
-%if (0%{?fedora} >= 29 || 0%{?rhel} >= 8)
```

Comment:
So we assume that this change will not ever get put into the RHEL7 branches?

Reply:
Yeah, this commit can never go to rhel7 branch.